tests/main/fakestore-install/task.yaml

```diff
@@ -20,6 +20,10 @@ execute: |
 
     snap ack "$TESTSLIB/assertions/testrootorg-store.account-key"
 
+    # ensure that "core" is installed its a prereq of the base
```

Comment:
Sentence isn't reading properly. Maybe

    # Ensure that "core" is installed as it's a pre-req of the base but ...

if that's what it means?

Reply:
Yes, I think so. Applied.